homeassistant/components/google_tasks/quality_scale.yaml

```diff
@@ -0,0 +1,76 @@
+rules:
+  # Bronze
+  config-flow: done
+  brands: done
+  dependency-transparency: done
+  common-modules:
+    status: done
+    comment: |
+      The integration has a coordinator.py and no base entities.
+  has-entity-name: done
+  action-setup:
+    status: done
+    comment: The integration does not register any actions.
+  appropriate-polling: done
+  test-before-configure: done
+  entity-event-setup:
+    status: exempt
+    comment: Integration does not subscribe to events.
+  unique-config-entry: done
+  entity-unique-id: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  test-before-setup:
+    status: todo
+    comment: |
+      The integration refreshes the access token, but does not poll the API. The
+      setup can be changed to request the list of todo lists in setup instead
+      of during platform setup.
+  docs-high-level-description: done
+  config-flow-test-coverage: done
+  docs-actions: done
+  runtime-data:
+    status: todo
+    comment: |
+      The integration currently stores config entry data in `hass.data` and
+      needs to be moved to `runtime_data`.
+
+  # Silver
+  log-when-unavailable: todo
```

Comment:
This is already handled by the coordinator, right?

Reply:
Updated.